linux/action.sh

```diff
@@ -17,8 +17,8 @@ REPO_VERSION=${REPO_VERSION/v/}
 if [[ -z $POST_INSTALL ]]; then
     POST_INSTALL="
     test -e /etc/newrelic-infra/integrations.d/${INTEGRATION/nri-/}-config.yml.sample
-    test -x /var/db/newrelic-infra/newrelic-integrations/bin/${INTEGRATION}
-    /var/db/newrelic-infra/newrelic-integrations/bin/${INTEGRATION} -show_version 2>&1 | grep -e $TAG
+    test -x /opt/newrelic-infra/newrelic-integrations/bin/${INTEGRATION}
+    /opt/newrelic-infra/newrelic-integrations/bin/${INTEGRATION} -show_version 2>&1 | grep -e $TAG
```

Comment:
can we support old and new structures? Otherwise it would be a breaking change (maybe we want that to keep track of what migrated)